src/work-allocation/containers/task-list/task-list.component.html

```diff
@@ -1,11 +1,19 @@
+<div *ngIf="pagination" class="pagination-top" attr.aria-label="{{ tasksTotal }} results have been found" role="status">
+  <span class="text-16" id="search-result-summary__text">Displaying <span class="govuk-!-font-weight-bold">{{
+      tasks?.length || 0 }}</span> of <span class="govuk-!-font-weight-bold">{{ tasksTotal }} </span>tasks</span>
```

Comment:
Shouldn't this incorporate the pagination somewhere along the line, seeing as that what this PR seems chiefly to be about?
`Displaying 10 of 23 tasks` says nothing about which page you're on.
`Displaying 3 of 23 tasks` could actually be quite confusing.

Elsewhere in the app, we (quite reasonably) render this sort of thing as:
`Displaying 1-10 of 23 tasks`
`Displaying 21-23 of 23 tasks`

Reply:
Yes I have raised this with both @rayliangatos  and Aleena and they are comfortable with the format

Reply:
Even though it's inconsistent with the way the case list works? Is the case list going to _also_ be changed to make it more confusing?

Reply:
This is strictly following the prototype in work allocation.

Reply:
I would like to bring in Aleena on this who is not on Github